src/View/Components/BaseBadge.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace WireUi\View\Components;
+
+abstract class BaseBadge extends Component
+{
+    public string $pulseColor = 'bg-slate-500';
+
+    public string $pulsePingColor = 'bg-slate-400';
+
+    public function __construct(
+        public bool $info = false,
+        public bool $warning = false,
+        public bool $positive = false,
+        public bool $negative = false,
+        public bool $dismissible = false,
```

Comment:
It's a slot? If so, remove it, don't need to add the slot into the construct arguments, then update: 
```diff
-  @if ($dismissible) 
+  @isset ($dismissible) 
```